TableModuleTemplate/<%module%> Module/UserInterface/Wireframe/<%prefix%><%module%>Wireframe.m

```diff
@@ -21,7 +21,7 @@ @interface <%prefix%><%module%>Wireframe ()
 
 @implementation <%prefix%><%module%>Wireframe
 
-- (void)presentTableControllerFromNavigationController:(UINavigationController *)nc
```

Comment:
Add additional methods - presentFromNavigation: presentModalOn: presentFromWindow: and handle this situations in dismiss